metabase/metabase
# fix hiding incorrect y-axis on hover

Closes https://github.com/metabase/metabase/issues/49874

### Description

On line/area/bar/combo charts with two axis when a series hovered we should hide the other y-axis not related to the series.

### How to verify

- Create a line chart with two y-axes
- Hover each series and ensure it keeps only 1 axis that is related to the series visible

### Checklist
